magento/inventory
# MSI-891: Stock status title is visible on "Advanced Inventory" in multi source mode

### Fixed Issues (if relevant)

1. https://github.com/magento-engcom/msi/issues/891

### Manual testing scenarios
1. Create one more source.
2. Open simple product creation.

Expected result:
- Stock status field is fully unvisible
